test_unstructured_ingest/expected-structured-output/local-single-file-with-encoding/fake-html-cp1252.html.json

```diff
@@ -19,7 +22,10 @@
       "data_source": {},
       "filetype": "text/html",
       "languages": [
-        "eng"
+        "por",
+        "cat",
+        "eng",
+        "vie"
```

Comment:
Though I'm confused how this one isn't being detected as English given the length check is `<20` and this is 19 characters...

Reply:
I think something might still be wrong with the ingest... This really shouldn't be happening.

Reply:
Unless this is part of a larger text document that is primarily Latin or some other language. The default is to detect language at the _document_ level rather than the _element_ level. So if a document is primarily in Latin, every element will be detected as such unless `detect_language_per_element` is set to `True`